pubspec.yaml

```diff
@@ -7,13 +7,12 @@ version: 2.0.3
 dependencies:
   flutter:
     sdk: flutter
+  flutter_test:
+    sdk: flutter  
```

Comment:
This has to get moved to another package. We can't have the runtime depending on flutter_test.

Reply:
Sorry for the late response.
I see what you mean. 
How should we go further?
Should I create  a separate  repository or should we integrate the package in this one?
For example like this:
```
flutter_svg
│ 
├──.github
├──.gitignore
├──READMED.md
├──LICENSE
├──......
└─packages
       │ 
       ├── flutter_svg
       │         ├── example
       │         ├── lib
       │         ├── test
       │         ├── README.md
       │         ├── ....
       │         └── pubspec.yaml
       │ 
       └── flutter_svg_test
                   ├── example
                   ├── lib
                   ├── test
                   ├── README.md
                   ├── ....
                   └── pubspec.yaml
```
I am willing to make this work, but you probably need to guide me  a bit.

I can add some more finder like the following one, so the package is not that small.
```
/// Finds widgets created by [SvgPicture.asset] with the [path] argument.
///
/// ## Sample code
/// ```dart
/// expect(svgAssetWithPath('assets/asset_name.svg'), findsOneWidget);
/// ```
/// This will match one [SvgPicture.asset] with the [path] 'assets/asset_name.svg'.
///
/// If the `skipOffstage` argument is true (the default), then this skips
/// nodes that are [Offstage] or that are from inactive [Route]s.
Finder svgAssetWithPath(String path, {bool skipOffstage = true}) {
  return _SvgAssetWithPathFinder(svgPath: path, skipOffstage: skipOffstage);
}

class _SvgAssetWithPathFinder extends MatchFinder {
  _SvgAssetWithPathFinder({required String svgPath, super.skipOffstage})
      : _svgPath = svgPath;

  final String _svgPath;

  @override
  String get description => "Path: '$_svgPath' and created by SvgPicture.asset";

  @override
  bool matches(Element candidate) {
    var result = false;
    final widget = candidate.widget;
    if (widget is SvgPicture) {
      final bytesLoader = widget.bytesLoader;
      if (bytesLoader is SvgAssetLoader) {
        result = bytesLoader.assetName == _svgPath;
      }
    }
    return result;
  }
}
```

Reply:
Yes, this needs to be its own package, which the repo isn't currently structured well for. I think your proposed structure is fine.